creator-node/src/services/stateMachineManager/stateMonitoring/monitorState.jobProcessor.js

```diff
@@ -184,38 +165,80 @@ module.exports = async function (
     }
   } catch (e) {
     logger.info(`processStateMonitoringJob ERROR: ${e.toString()}`)
-    jobFailed = true
+    jobSucceeded = false
   } finally {
-    _addToDecisionTree(decisionTree, jobId, 'END processStateMachineOperation')
+    _addToDecisionTree(decisionTree, 'END processStateMachineOperation', logger)
 
     // Log decision tree
-    _printDecisionTree(decisionTree, jobId)
+    _printDecisionTree(decisionTree, logger)
   }
 
   // The next job should start processing where this one ended or loop back around to the first user
   const lastProcessedUser = users[users.length - 1] || {
     user_id: 0
   }
   return {
-    lastProcessedUserId: lastProcessedUser?.user_id || 0,
-    jobFailed,
-    users,
-    unhealthyPeers,
-    replicaSetNodesToUserClockStatusesMap,
-    userSecondarySyncMetricsMap
+    jobsToEnqueue: {
```

Comment:
if we are standardizing this return format, might be good to make a util function / object interface or something across all the jobs